src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/AlwaysEncryptedHelperClasses.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Data.SqlClient
     /// <para> Represents a single encrypted value for a CEK. It contains the encrypted CEK,
     ///  the store type, name,the key path and encryption algorithm.</para>
     /// </summary>
-    internal class SqlEncryptionKeyInfo
+    internal sealed class SqlEncryptionKeyInfo
```

Comment:
Sealing an internal class, while it may offer some performance improvements, can make testing more difficult. Given this, we opted to use `InternalsVisibleToAttribute` for some of our classes to facilitate testing. Is there any specific reasoning behind it?

Reply:
There's no specific rationale beyond performance - it's on the hot path, so I usually start off sealing non-inherited classes then unsealing them as appropriate. I'll revert the sealing (and one other field rename which I missed earlier.)